e2e-tests/commands/insurance/declarations/complete-modern-slavery-form.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * completeModernSlaveryForm
+ * complete the "modern slavery" form.
+ * @param {Boolean} willAdhereToAllRequirements: radio answer
+ * @param {Boolean} hasNoOffensesOrInvestigations: radio answer
+ * @param {Boolean} isNotAwareOfExistingSlavery: radio answer
+ * @param {String} conditionalFields.cannotAdhereToAllRequirements: Textarea answer
+ * @param {String} conditionalFields.offensesOrInvestigations: Textarea answer
+ * @param {String} conditionalFields.awareOfExistingSlavery: Textarea answer
+ */
+const completeModernSlaveryForm = ({
+  willAdhereToAllRequirements = true,
+  hasNoOffensesOrInvestigations = true,
+  isNotAwareOfExistingSlavery = true,
+  conditionalFields = {
+    cannotAdhereToAllRequirements: '',
+    offensesOrInvestigations: '',
+    awareOfExistingSlavery: '',
+  },
+}) => {
+  if (willAdhereToAllRequirements) {
+    cy.clickYesRadioInput(0);
+  }
+
+  if (willAdhereToAllRequirements === false) {
+    cy.clickNoRadioInput(0);
+  }
+
+  if (hasNoOffensesOrInvestigations) {
```

Comment:
would switch be better here?

Reply:
It wouldn't work because in most scenarios we need at least 3x of these commands to run, not a single condition.